vllm/engine/arg_utils.py

```diff
@@ -435,6 +457,16 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             'This should be a JSON string that will be '
                             'parsed into a dictionary. Ignored if '
                             'tokenizer_pool_size is 0.')
+
+        # Multimodal related configs
+        parser.add_argument('--limit-mm-per-prompt',
+                            type=nullable_kvs,
+                            default=EngineArgs.limit_mm_per_prompt,
+                            help='For each multimodal plugin, limit how many '
+                            'inputs to allow for each prompt. Expects a '
+                            'comma-separated list of items, e.g.: '
+                            '`KEY1=VALUE1,KEY2=VALUE2`')
```

Comment:
[Some models](https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/chat_utils.py#L128) do not include placeholders, so imo it would be more appropriate to define it in terms of number of image instances inputted.

Reply:
I have updated the help text, see if it is clearer to you now.